### ISSUE TITLE
Office in Seattle for Nomadic RAM's

### ISSUE THREAD

**Issue by lapin7** [rchain/bounties#582]
## Intro

> The Board discussed office space needs and @leithaus updated the board on recent
search activities for office space. In particular, @leithaus expressed an interest in purchasing
a West Seattle residential property to use as a developer office, in addition to the [Pioneer Square](https://www.pioneersquare.org/)
location in contemplation. @leithaus is directed to purchase [the house](https://www.matrix.nwmls.com/Matrix/Public/Portal.aspx?ID=23612940236) for use as an RChain office.

- [x] Describe the job of House Manager
- [ ] Find a person for the position House Manager
- [x] Set up rules for booking
- [x] Set up rules for using the space
- [ ] Set up maintenance
- [ ] Add SMART issues to this epic
- [ ] Suggested budget $2000
- [ ] Duration 2 weeks
- [ ] ETC.

**Comment by jplepel** [comment]
I have put together a google doc. Feel free to comment and add suggestions. 

https://docs.google.com/document/d/1TlX3QrBrOu6hpLskKef3V9u5PG3MkCNe_blTJ5EehpY/edit?usp=sharing

**Comment by Tonyprisca13** [comment]
The doc seem ok

**Comment by jplepel** [comment]
@Tonyprisca13 please add more, or some comments. Sure I missed some points.

**Comment by Ojimadu** [comment]
@jplepel Have a look at my suggestions and edits.

**Comment by jplepel** [comment]
@Ojimadu Good reference to the RTA link. I think one important thing to cover is tenant privacy due to the nature of the work being done. Tenants will want to feel they can leave the house without their electronics and documents being tampered. 

I added "Tenants and HM agree on room access during stay"... something as simple as a "Do not disturb" sign at a hotel letting the HM know that your room is not to be entered unless an extreme circumstance. For fun, we could test out LifeID when the time comes to allow access into certain rooms of the house.

**Comment by Ojimadu** [comment]
I think I made the doc kind of messy with the suggestion -was made on mobile-. @jplepel Please reject the suggestions so it can clear up.

**Comment by jplepel** [comment]
Do we plan on having the HM be a full time resident or someone nearby on call that can devote ~20-30hrs per week?

**Comment by ResonanceXX** [comment]
There has to adequate provision for the wage and/or salary demands of a full time HM, as opposed to a part time/ partial staff.
Then we could also consider the time demand for the role, how much of his/her presence is required to ensure things run smoothly in the office.

**Comment by jplepel** [comment]
I have cleaned up the doc. Have 3 options for the HM

-Full-time in home resident
-Full-time local resident 
-Part-time local resident

Full time HM position is responsible for all house work including housekeeping and lawn care. Free time is to be used as an assistant to board members.

**Comment by Tonyprisca13** [comment]
@jplepel we should go with full time HM. Having someone who is always available to meet domestic and other needs will help the RChain staff to have full focus on their work.

**Comment by jplepel** [comment]
@Tonyprisca13 , I think it'd be beneficial for a full time HM. Up to the board to see how "hands on/present" they want the HM to be.

**Comment by lapin7** [comment]
Updated doc https://docs.google.com/document/d/1TlX3QrBrOu6hpLskKef3V9u5PG3MkCNe_blTJ5EehpY/edit?usp=sharing

Now let's start recruitment of a full time HM.

**Comment by Ojimadu** [comment]
I guess we have to take this to Bill Swan (HRM) or Kenny Rowe so the recruitment can be advertised. I have sent a message to Kenny on discord but am not aware Bill Swan's discord ID.

**Comment by swanzer** [comment]
All - as I posted to discord a little earlier today, the enthusiasm and participation is great! Here is what I posted to discord this afternoon.  You were right in trying to reach me with the idea to post the job description before doing so.  What you created will be helpful; as I state below, its important to get input on job descriptions so they are accurate to actuals of the job.  So thanks for that.  
Message on discord today: 
Bill: - Today at 4:49 PM
Thanks @ysgjay: and @Lawrence: for offering to help with the ideas on managing the new house in Seattle. (from your messages on April 21st)   The enthusiasm is appreciated!  Yet we need to pull back on this a bit.  Some alternate ideas surfaced while at the Dev Conference which @kennyrowe:  and I have just started to discuss; possibly less of a 'live in' house manager concept and more of contracting specific services; cleaning, yard care, maintenance with some property management oversight.  More of an Air BnB concept.  As we get clearer on the locations use and envision more of its utilization potential, we have a better understanding on how it will be used every day.  We want to research this further.  Also, I'll just add this here and will post other places too; as Head of Human Resources for the Coop, any job description for an Employee of the Coop is going to need to be confirmed by myself before it is posted for recruiting potential applicants.  I will always need input to be sure they accurately describe the roles but I need to be sure we never put anything out to the public that could ever put the Coop into any legal trouble or could be further edited for maximizing optimal candidate reach.   Let me know if you have any questions.  Thanks

**Comment by lapin7** [comment]
Hi @swanzer @kennyrowe 
I'm busy with improving collaboration and RChain's bounty program. My goal is to make github/rchain/bounties the main place for RChain's operation and management.

So any information about hrm, finance, marketing or whatever could be posted here.

> Some alternate ideas surfaced while at the Dev Conference which @kennyrowe: and I have just started to discuss; possibly less of a 'live in' house manager concept and more of contracting specific services; cleaning, yard care, maintenance with some property management oversight. More of an Air BnB concept. 

Can these discussions  of ideas between @kennyrowe and yourself ( @swanzer ) be posted as comments under this #582 issue?

That would be helpful for RAM's to guide their efforts.

Also the state of the sale is important. Because on the [brokers site](https://www.matrix.nwmls.com/Matrix/Public/Portal.aspx?ID=23612940236) the property is mentioned as **sold**.

**Comment by jplepel** [comment]
@lapin7 Thanks for the mention, I was considering the position. Will wait to hear more from the Board and @kennyrowe on a clearer direction of the HM role. I believe the 'live in' and 'air bnb' style setups are both acceptable, depends on how hands on our typical nomads would like the experience in the RHouse.

**Comment by swanzer** [comment]
Thanks @lapin7.  First, there are some legal considerations that need to be looked into further with the City of Seattle and taken into consideration.  Since it is a business utilizing the zoned residence, we may be required to:
- have business license registration for the property
- short term rental permit for offering residence to guests for periods shorter than a time period (30 or 60 days for example).
- Could be a requirement for notifying the neighborhood of the buildings use
- Possible Occupancy Permit requirements
- Could be tax registration and applicable taxes
- Possible City inspections for fire suppression and safety measures
I mention these items as certainly, we all want this opportunity to work and not run into anything that puts RChain Coop in a bad light with negative Press or publications.  Seattle being the Tech area it is, gives plenty of press coverage for any Tech organization and the public is never afraid to be very boisterous when pushing an issue or trying correct something the public is upset about.  We want to avoid this by approaching the RHouse as properly as possible.  So in this light, we look to envisioning the best possible RHouse. 
In regards specifically to the idea of a House Manager, whether FT or PT, Live-in or local-resident, or even Contracting with a Property Managing Service, the Coop will do well to consider what we uncover in legal requirements with the City of Seattle and then craft a function that will work best with those requirements and serve the needs of the Coop's intended use of the RHouse.  Personally, I gravitated towards the House Manager idea initially as well.  Then the above started to 'click' and someone from the Dev Conference (whom I forget now) suggested looking at the Air BnB model; online reservations, acknowledging compliance to rules and expectations, assuming responsibility while staying, etc, etc. and this sounded like something to be researched further before we got too far down the road of hiring someone, for it may be that what we need are services to maintain the property and less of someone to be present every day.  
Again, City requirements + the Coops vision + the daily use of the house all need to fit nicely together to avoid problems.  I hope this all helps.  Thanks
